src/app/css/_application.scss

```
@@ -38,7 +38,7 @@
     }
 }
 .app {
-    font-family: "Segoe UI" !important;
+    font-family: "Arial" !important;
```

```suggestion
    font-family: "Segoe UI", "Arial" !important;
```
To allow using Segoe UI if it is installed and Arial as a fallback